documentation/specs/Stepper.md

```diff
@@ -0,0 +1,102 @@
+# `Stepper` Component Specification
+
+## Overview
+
+A `Stepper` component is used to indicate progress as the user goes through a multi-step process.
+
+### Use Cases
+
+- Guide a user through a large form where parts of the form can be separated into logical steps.
+- Guide a user through a sequence of steps where the next step depends on data from the previous step.
+
+### Features
+
+- The `orientation` property can be used to display a vertical or horizontal stepper.
+- The `color` property can be used to display a stepper with primary or inverse colors; inverse is suitable on darker backgrounds.
+- The step sequence is non-linear, which means users can navigate to previously completed steps.
+- On small screens, the step label is not visible.
+- A step that is not completed or active is considered disabled.
```

Comment:
Does disabled mean they can't click it? Or purely for styling leaving the onPress handler to determine if that step should become active?
 
What about the concept of isAccessible? If there is Step A, B, and C. And a user gets to step C, doesn't complete it, but goes back to step A, it seems like they wouldn't be able to / doesn't look like they can click step C to go back to it in order to finish it.

If not wanting to have it explicitly handled, it seems like the step after the last "completed" step should be clickable/not disabled even though it's not completed or active.

Reply:
i think my original wording was quite confusing, i updated it and your comment made me realize i think it'd be good if the component was explicit about its status options. i added a prop `status` which replaces `isComplete` and `status` can be active, accessible, disabled, or completed. depending on how implementation goes, i may do away with the `activeStep` prop since it looks like `status` should be able to capture the meaningful info on its own